docs/_webCode/js/downloads.js

```diff
@@ -147,23 +149,15 @@ async function getVersion(build, cachedVersions, newCachedVersions) {
   return {version: version, downloadUrl: build["downloadUrl"]};
 }
 
-async function downloadWithRename(url, filename) {
-  return new Promise((resolve, reject) => {
-    const xhr = new XMLHttpRequest();
-    xhr.open("GET", url, true);
-    xhr.responseType = "blob";
-    xhr.onload = function () {
-      if (this.status === 200) {
-        const blob = this.response;
-        const link = document.createElement("a");
-        link.href = URL.createObjectURL(blob);
-        link.download = filename ? filename : url.split("/").pop();
-        link.click();
-        resolve();
-      } else {
-        reject();
-      }
-    };
-    xhr.send();
+function downloadWithRename(url, filename) {
+  return fetch(url).then(async response => {
+    if (!response.ok) {
+      log.error("Error while downloading file: " + response.status + " " + response.statusText + "")
```

Comment:
Missing Semicolon

Reply:
It would save me a lot of time if you made suggestions for these minor oversights.